### PR TITLE
Added an uncle sam meme

### PR DESCRIPTION
**Pull request by tenderlove**
merge me

### CODE REVIEW

**Comment by drbrain** [comment]
default to I Want You

**Comment by tenderlove** [comment]
updated

**Comment by leejarvis** [comment]
merge this or else

**Comment by drbrain** [comment]
applied

**Comment by josevalim** [comment]
SCUMBAG

**Comment by andion** [comment]
Boss Cat

**Comment by yonkeltron** [comment]
THIS IS HOW PULL REQUESTS WORK

**Comment by kennethreitz** [comment]
Closed?  

***comments anyway\***

**Comment by gtswagger** [comment]
</img>

**Comment by gtswagger** [comment]
</img>

**Comment by gtswagger** [comment]
</img>

**Comment by domdorn** [comment]
commenting anyway.

**Comment by ilude** [comment]
alt imma let you pull

**Comment by ilude** [comment]
Its fried eggs fried eggs

**Comment by eric** [comment]
@rtomayko any ideas?

File: lib/meme.rb [hunk 1]
```diff
@@ -52,6 +52,7 @@ class Meme
   GENERATORS['SWEDISH_CHEF']      = [186651, 'Swedish-Chef']
   GENERATORS['TOWNCRIER']         = [434537, 'Towncrier']
   GENERATORS['TROLLFACE']         = [1030,   'Troll-Face']
+  GENERATORS['US_POINT']          = [131083,   'Uncle-Sam-Point', 'I WANT YOU']
```
Comment (tenderlove) [inline lib/meme.rb:55]: @rtomayko

alignment bro

Comment (dstrelau) [inline lib/meme.rb:55]: @rtomayko



drbrain/meme@4e1ad6495d1b4b64b39cf3fe8ddc0706d61e33c6

**Pull request closed without merging**